src/v2/view-builder/views/idp/AuthenticatorIdPView.js

```diff
@@ -18,6 +18,29 @@ const Body = BaseForm.extend({
       : loc('oie.idp.enroll.description', 'login', [displayName]);
   },
 
+  showMessages () {
+    // IdP Authenticator error messages are not form errors
+    // Parse and display them here.
+    const messages = this.options.appState.get('messages') || {};
+    if (Array.isArray(messages.value)) {
+      this.add('<div class="ion-messages-container”></div>', '.o-form-error-container');
+
+      messages
+        .value
+        .forEach(messagesObj => {
+          const msg = messagesObj.message;
+          if (messagesObj.class === 'ERROR') {
+            this.add(createCallout({
```

Comment:
Suppose if there are multiple errors.. is this going to add multiple callouts ??

Reply:
Great question, and I believe it will.For this flow, we shouldn't have more than one error message - so I decided to copy the behavior used in other authenticators:
-  [Challenge Okta Verify (v2)](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js#L17-L37)
- [Terminal View](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/TerminalView.js#L54-L75)

As mentioned in [this comment](https://github.com/okta/okta-signin-widget/pull/1738/files/53a50163a5bcf39edb963e5a5f4d3ea672a54114#r592554832) I think there is value in moving all error logic to a single handler as a part of a `BaseAuthenticatorView`. When we do this, we can loop in UX to see if we should concatenate all the error messages together, or keep this separate as we're currently doing.